run.sh

```diff
@@ -252,6 +256,31 @@ fi
 # Validate genesis
 [ "$VALIDATE_GENESIS" == "1" ] && $PROJECT_BIN validate-genesis
 
+# Cosmovisor
+if [ "$COSMOVISOR_ENABLED" == "1" ]; then
```

Comment:
Wondering whether we should move this section to the Dockerfile so Cosmovisor is pre-built and doesn't have to be installed at runtime. On the one hand it would save some start-up time at the expense of being version flexible for the user. It also puts the onus on us to keep it up to date.

Is it typical to use any version other than the latest Cosmovisor?

Reply:
Well if the version is "stable enough" (which I have no idea how to define)
We can just embed it
Problem is we should still allow custom cosmovisor version to be used (in case people want to test new versions)
But that makes it a bit complicated...

I would like this to be merged, used (which need to be enabled explicitly) and problems reported back (hopefully none). Further improvements can be introduced later (if people actively ask for it then it's even better).

(Otherwise I need to keep using my custom images to use Cosmovisor 😓)

Reply:
Sounds good to me. Including it in the Dockerfile also unnecessarily bloats the images for users who don't need Cosmovisor, so I think this route is sensible.

I'll merge in a release this week, probably Wednesday.